src/blocks/common.blocks/block-example/block-example.js

```diff
@@ -1,6 +1,6 @@
-modules.define('i-bem__dom', ['jquery'], function (provide, $, BEMDOM) {
+modules.define('block-example', ['i-bem__dom', 'jquery', 'dropdown'], function (provide, BEMDOM, $, Dropdown) {
 
-BEMDOM.decl('block-example', {
+BEMDOM.decl(this.name, {
```

Comment:
nope, it's still there at https://github.com/bem/bem-site-engine/blob/qr%40dev/src/blocks/common.blocks/block-example/block-example.js#L113

Actually we need to update the way modules are defined everywhere on bem-site-engine.